### PR TITLE
Bring-Example Integration tests

### PR DESCRIPTION
**Pull request by Talakh**

### CODE REVIEW

File: bring-examples/src/test/java/com/bobocode/petros/bring/service/ServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import com.bobocode.petros.bring.exception.NoSuchBeanException;
+import com.bobocode.petros.bring.exception.NoUniqueBeanException;
+import com.bobocode.petros.bring.service.impl.MorningServiceImpl;
+import org.junit.jupiter.api.Test;
+
+import static com.bobocode.petros.bring.TestData.CONTEXT;
+import static org.junit.jupiter.api.Assertions.*;
+
+class ServiceTest {
```
Comment (vromantsev) [inline bring-examples/src/test/java/com/bobocode/petros/bring/service/ServiceTest.java:11]: There are some tests failures, could you please fix the build?

Comment (Talakh) [reply bring-examples/src/test/java/com/bobocode/petros/bring/service/ServiceTest.java:11]: It requires aware mechanism and fixes for the component scanner, waiting for merge this 2 tasks

Comment (vromantsev) [reply bring-examples/src/test/java/com/bobocode/petros/bring/service/ServiceTest.java:11]: got it. I prepared PR where a bug with @Repository and @Service annotations is fixed. Scanner improvements will be provided by @crezione1

Comment (vromantsev) [reply bring-examples/src/test/java/com/bobocode/petros/bring/service/ServiceTest.java:11]: @Talakh aware mechanism is implemented. You can try adapting it here.

**Review by vromantsev — APPROVED** [review]

**Review by Shtramak — APPROVED** [review]

**Pull request merged**